stable/redis-ha/templates/redis-sentinel-deployment.yaml

```diff
@@ -23,6 +23,29 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       {{- end }}
+      {{- if eq .Values.antiAffinity "hard" }}
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                app: {{ template "redis-ha.name" . }}
+                release: {{ .Release.Name }}
+                component: sentinel
```

Comment:
That's correct. The intent of this PR is to prevent multiple sentinel pods or multiple server pods from running on the same nodes, so sentinels have antiaffinity to other sentinels and servers have antiaffinity to other servers.

I don't see any requirements on placement of sentinels vs. servers in [the Sentinel documentation](https://redis.io/topics/sentinel) so I believe this is sufficient.